app/src/main/java/org/mozilla/focus/shortcut/IconGenerator.kt

```diff
@@ -55,6 +57,28 @@ class IconGenerator {
             return drawCharacterOnBitmap(context, character, shape)
         }
 
+        @JvmStatic
+        fun generateSearchEngineIcon(context: Context): Bitmap {
+            val options = BitmapFactory.Options()
+            options.inMutable = true
+            val shape = BitmapFactory.decodeResource(context.resources, R.drawable.ic_search_engine_shape, options)
+            val frameSize = 0.15
```

Comment:
\[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.